Vagrantfile

```diff
@@ -113,7 +113,7 @@ Vagrant.configure(2) do |config|
   end
 
   config.vm.provider "virtualbox" do |vb|
-    vb.memory = "3196"
+    vb.memory = "6144"
```

Comment:
Does the following commit solve the issue:
https://github.com/nlesc-sherlock/emma/commit/f1ca5f460725b47a47dbd7d35e36e5b2113bc92a